### DISCUSSION TITLE
Metadata file is case sensitive, switch to not sensitive?

### DISCUSSION THREAD

**Discussion by Droppisalt** [meisnate12/Plex-Meta-Manager#1050] [Q&A]
Is there any way to prevent PMM to look through a metadata file with case sensitivity? E.g: 
```
"Behind the Monsters":
    sort_title: "Behind the Monsters"
    year: 2021
    url_poster: https://theposterdb.com/api/assets/242810
    season_display: hide
```
Gets ignored because in Plex it's "Behind The Monsters", but as soon as I change the metadata file to "Behind The Monsters", it works

**Comment by meisnate12** [comment]
if its displayed in your plex as `the` and not `The` then i would recommend performing a plex dance on the item

**Reply by Droppisalt** [reply]
Totally right. I should probably do that!
